descriptions/ab+/en_us.lua

```diff
@@ -656,7 +656,7 @@ EID.descriptions["en_us"].trinkets={
 	{"85", "Karma", "Using a Donation Machine can heal you or spawn a beggar"},
 	{"86", "Lil Larva", "Destroying poop spawns a blue fly"},
 	{"87", "Mom's Locket", "Heal half a heart when using keys#Turns half hearts into full hearts"},
-	{"88", "NO!", "Very low chance for spacebar items in {{TreasureRoom}}Treasure Rooms"},
+	{"88", "NO!", "Prevents active items from being generated in all item pools},
```

Comment:
Why is this change done in the ab+ descriptions file? Was it incorrect before or did you just put the new repentance description in here?

Reply:
It was incorrect before, the trinket didn't have any updates in Repentance.